skimage/measure/_regionprops.py

```diff
@@ -358,22 +358,73 @@ def __eq__(self, other):
         return True
 
 
+def to_dict(regions, want=['label', 'bbox'], separator='-', cancel_auto=False):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        _RegionProperties
+    want : list, optional
+        str of a property the user wants to include
+    separator : str, optional
+        used in joining coordinates
+    cancel_auto : bool, optional
+        prevents autmatic adding of label and bbox
+
+
+    Output
+    ------
+    A dictionary with properties in accordance to the wanted properties
+
+    Examples
+    --------
+    >>> from skimage import data, util
+    >>> from skimage.measure import label
+    >>> from pandas.DataFrame import from_dict
+    >>> img = util.img_as_ubyte(data.coins()) > 110
+    >>> label_img = label(img, connectivity=img.ndim)
+    >>> props = regionprops(label_img)
+    >>> out = from_dict(to_dict(props))
+
+
+"""
+
+    objects = {'image', 'coords', 'convex_image', 'filled_image',
+               'intensity_image'}
+
+    if not cancel_auto and 'label' not in want:
+        want.insert(0, 'label')
+    if not cancel_auto and 'bbox' not in want:
+        want.insert(0, 'bbox')
```

Comment:
Label should be added by default without the flag, but I think bbox is not essential if the user doesn't ask for it. So:

```python
if always_use_label and 'label' not in properties:
    properties = ['label'] + list(properties)
```